qemu/tests/nvdimm.py

```diff
@@ -37,13 +39,14 @@ def run_guest_cmd(self, cmd, check_status=True):
             self.test.fail("Execute command '%s' failed, output: %s" % (cmd, output))
         return output.strip()
 
-    def verify_nvdimm(self, vm):
+    def verify_nvdimm(self, vm, mems):
         """
         verify nvdimm in monitor and guest
 
         :params vm: VM object
+        :params mems: memory objects
         """
-        dimms_expect = set(["dimm-%s" % dev for dev in self.params.objects("mem_devs")])
+        dimms_expect = set("dimm-%s" % mem for mem in mems)
```

Comment:
Suggest to print the dimms_expect, to avoid the issue before: test pass, debug log is normal but miss one check point.

Reply:
Done, thanks.